packages.yml

```
@@ -2,5 +2,8 @@ packages:
 - package: fivetran/shopify
   version: [">=0.11.0", "<0.14.0"]
 
-- package: fivetran/klaviyo
-  version: [">=0.7.0", "<0.8.0"]
\ No newline at end of file
+# - package: fivetran/klaviyo
+#   version: [">=0.7.0", "<0.8.0"]
+- git: https://github.com/fivetran/dbt_klaviyo.git
+  revision: bugfix/too-many-partitions
+  warn-unpinned: false 
```

To update before merge
```suggestion
 - package: fivetran/klaviyo
   version: [">=0.8.0", "<0.9.0"]
```